admin/src/pages/MailingList.tsx

```
@@ -87,6 +100,21 @@ const MailingListPage: React.FC = () => {
     retry: noRetryOnClientError,
   })
 
+  const toggleSelectAllOnPage = useCallback(() => {
+    if (!preview?.rows) return
+    const pageIds = preview.rows.map((r) => r.id)
+    setSelectedUserIds((prev) => {
+      const allSelected = pageIds.every((id) => prev.has(id))
+      const next = new Set(prev)
+      if (allSelected) {
+        pageIds.forEach((id) => next.delete(id))
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)